### PR TITLE
remove paddle.fluid.layers.layer_norm

### PR DESCRIPTION
**Pull request by zhengqiwen1997**
### PR types

Others
### PR changes

APIs
### Describe

To clean the code in paddle.fluid, the useless codes should be removed and code still be used should move to a new position.

For function paddle.fluid.layers.layer_norm, since it is not used in version 2.x, should be removed.

For unittest code where this function is used, if it is aimed to test the old API, it can be removed directly. Otherwise, the paddle.fluid.layers.layer_norm should be replaced with paddle.static.nn.layer_norm.

### CODE REVIEW

**Review by zoooo0820 — DISMISSED** [review]
LGTM

**Review by jeff41404 — DISMISSED** [review]
LGTM

File: python/paddle/static/nn/common.py [hunk 1]
```diff
@@ -3234,3 +3236,139 @@ def py_func_demo():
...
+    shift=True,
+    begin_norm_axis=1,
+    epsilon=1e-05,
+    param_attr=None,
+    bias_attr=None,
+    act=None,
+    name=None,
+):
+    r"""
+    :api_attr: Static Graph
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py]: 这个删掉吧

File: python/paddle/static/nn/common.py [hunk 2]
```diff
@@ -3234,3 +3236,139 @@ def py_func_demo():
...
+    **Layer Normalization Layer**
+
+    The API implements the function of the Layer Normalization Layer and can be applied to mini-batch input data.
+    Refer to `Layer Normalization <https://arxiv.org/pdf/1607.06450v1.pdf>`_
+
+    The formula is as follows:
+
+    ..  math::
+
+        \\mu & = \\frac{1}{H}\\sum_{i=1}^{H} x_i
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py]:
```suggestion
        \mu & = \frac{1}{H}\sum_{i=1}^{H} x_i
```

Comment (Ligoml) [reply python/paddle/static/nn/common.py]: 下面的公式也改一下吧

image

File: python/paddle/static/nn/common.py [hunk 3]
```diff
@@ -3234,3 +3236,139 @@ def py_func_demo():
...
+            a default :code:`ParamAttr` would be added as scale. The
+            :attr:`param_attr` is initialized as 1 if it is added. Default: None.
+        bias_attr(ParamAttr, optional): The parameter attribute for the learnable
+            bias :math:`b`. If :attr:`shift` is False, :attr:`bias_attr` is
+            omitted. If :attr:`shift` is True and :attr:`param_attr` is None,
+            a default :code:`ParamAttr` would be added as bias. The
+            :attr:`bias_attr` is initialized as 0 if it is added. Default: None.
+        act(str, optional): Activation to be applied to the output of layer normalization.
+                  Default: None.
+        name(str): The default value is None.  Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name` .
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py]:
```suggestion
        name(str, optional): The default value is None.  Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name` .
```

**Review by Ligoml — APPROVED** [review]
LGTM for docs

**Review by jeff41404 — APPROVED** [review]

**Review by wanghuancoder — APPROVED** [review]

**Pull request merged**